File: Client/src/api/dto/SignInResponseDto.js
export default function createSignInResponseDto ({isRegistrationSuccessful, errors, accessToken, refreshToken, applicationUserDto}) {
    return {
        isRegistrationSuccessful: isRegistrationSuccessful,
        errors: errors,
        accessToken: accessToken,
        refreshToken: refreshToken,
        applicationUserDto: applicationUserDto
    };
}

File: Server/BusinessLogic/DTOs/SignInResponseDTO.cs
using System.ComponentModel.DataAnnotations;

namespace BusinessLogic.DTOs;

public class SignInResponseDTO
{
    public bool IsSuccessful { get; set; }
    public IEnumerable<string> Errors { get; set; }
    public string AccessToken { get; set; }
    public string RefreshToken { get; set; }
    public ApplicationUserDTO ApplicationUserDTO { get; set; }
}